src/shapes/Circle.java

```diff
@@ -0,0 +1,18 @@
+package shapes;
+
+public class Circle implements Shape {
+
```

Comment:
define a field to store a radius
for example
`private int radius`

Set it later on in the constructor
`public Circle(int radius) {
   this radius = radius;
}`